<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Character</class>
 <widget class="QWidget" name="Character">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/img/hero.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 22pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>QWQ大冒险</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bb1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb11">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bb10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="HP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="ATK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ATK </string>
   </property>
  </widget>
  <widget class="QLabel" name="L1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>伙伴1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Bauhaus 93&quot;;</string>
   </property>
   <property name="text">
    <string>你的仓库</string>
   </property>
  </widget>
  <widget class="QLabel" name="L2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>伙伴2：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zb1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="zb2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="d1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="d2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>AP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="E1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/AP.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="E2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/AP.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="E3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/AP.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
